README.md

```diff
@@ -75,7 +79,8 @@ As of now, these are the supported resources:
    usage: commercetools-project-sync
     -h,--help               Print help information.
     -s,--sync <arg>         Choose one of the following modules to run: "types", "productTypes", "states", "categories", 
-                            "cartDiscounts", "products", "inventoryEntries" or "all" (will run all the modules).
```

Comment:
I would follow the order and put tax categories after the states.
`Choose one of the following modules to run: "types", "productTypes", "states", "taxCategories", "categories"... `

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/139/commits/f56b41e6d35e6d2753f27a8f25d948e335c2e4de